simulator/func_sim/operation.h

```diff
@@ -31,7 +31,8 @@ enum OperationType : uint8
     OUT_STORE,
     OUT_J_JUMP,
     OUT_J_SPECIAL,
-    OUT_UNKNOWN
+    OUT_UNKNOWN,
```

Comment:
Usually it's better to keep the "undefined" value last. For instance, you can have a char array defined as:

const char* operation_names[OUT_LAST] = { ... , "Store", "Direct Jump", "Special Jump", "Floating Point"};